*** Makefile.orig	2015-04-21 15:12:48.000000000 -0700
--- Makefile	2015-04-21 15:16:34.000000000 -0700
***************
*** 24,29 ****
--- 24,30 ----
  lime-objs := tcp.o disk.o main.o
  
  KVER ?= $(shell uname -r)
+ CPU := $(shell uname -m)
  
  PWD := $(shell pwd)
  
***************
*** 32,38 ****
  default:
  	$(MAKE) -C /lib/modules/$(KVER)/build M=$(PWD) modules
  	strip --strip-unneeded lime.ko
! 	mv lime.ko lime-$(KVER).ko
  
  debug:
  	KCFLAGS="-DLIME_DEBUG" $(MAKE) -C /lib/modules/$(KVER)/build M=$(PWD) modules 
--- 33,39 ----
  default:
  	$(MAKE) -C /lib/modules/$(KVER)/build M=$(PWD) modules
  	strip --strip-unneeded lime.ko
! 	mv lime.ko lime-$(KVER)-$(CPU).ko
  
  debug:
  	KCFLAGS="-DLIME_DEBUG" $(MAKE) -C /lib/modules/$(KVER)/build M=$(PWD) modules 
